### ISSUE TITLE
Support clojure files (clj, cljs, cljc, edn)

### ISSUE THREAD

**Issue by yuhan0** [sbarex/SourceCodeSyntaxHighlight#70]
Currently only `.clj` files are recognized, but under the wrong syntax "Lisp Source File".

Ideally it should be classified as its own UTI and support the extensions `clj`, `cljc`, `cljs` and `edn`, which all share the same syntax.
 
Thanks!

**Comment by yuhan0** [comment]
Here's the mdls output for

clj
```
kMDItemContentType     = "dyn.ah62d4rv4ge80g5dn"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80g5dn",
    "public.data"
)
```
cljs
```
kMDItemContentType     = "dyn.ah62d4rv4ge80g5dnsq"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80g5dnsq",
    "public.data"
)
```


cljc:
```
kMDItemContentType     = "dyn.ah62d4rv4ge80g5dnqq"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80g5dnqq",
    "public.data"
)
```

edn:
```
kMDItemContentType     = "com.adobe.edn"
kMDItemContentTypeTree = (
    "com.adobe.edn",
    "public.data",
    "public.item"
)
```


It appears that the `.edn` extension is also a format for some obscure "Adobe DRM Activation Key": https://www.filesuffix.com/en/extension/edn, however in the context of code highlighting it is much more likely to be Clojure's extensible data notation (similar to JSON ~ Javascript)

**Comment by sbarex** [comment]
Support added on 2.0.1 release.

**Comment by yuhan0** [comment]
Hi, thanks for the fix! It looks like you missed out adding the md type for `.edn` files, although the changelog says so?
https://github.com/sbarex/SourceCodeSyntaxHighlight/commit/8664462d49f60459002224b5f16104c3dcaa35cc

**Comment by sbarex** [comment]
I am not clear about the type of syntax used by `.edn`. `Highlight` recognizes only the `.clj` extension. Are `.edn` files follow the clojure syntax or are they something else?

**Comment by yuhan0** [comment]
Yes, they're essentially the same syntax and can be highlighted the same way. 
https://learnxinyminutes.com/docs/edn/
https://learnxinyminutes.com/docs/clojure/

(like all lisps, Clojure's syntax is simply data structures, so data serialisation looks the same as code)

**Comment by yuhan0** [comment]
I'm using the latest 2.0.3 version, and edn files do not display in quicklook, whereas clj and other source files do:

![image](https://user-images.githubusercontent.com/22216124/117543937-985cf300-b051-11eb-82fa-31ec203778e6.png)

(the screenshot shows a `deps.edn` configuration file similar to `package.json` in Javascript development, it would be very handy to use Quicklook to preview a project's dependencies)

I only see the UTIs for clj, cljs and cljc in the commit 
https://github.com/sbarex/SourceCodeSyntaxHighlight/commit/8664462d49f60459002224b5f16104c3dcaa35cc#diff-2b400b188e3b7a9751610cf31d508012e7653259f342179445d0da588bd01897R199


As mentioned above, the UTI for `edn` is claimed by adobe for some DRM Activation Key that I couldn't find any information about:
```
kMDItemContentType     = "com.adobe.edn"
kMDItemContentTypeTree = (
    "com.adobe.edn",
    "public.data",
    "public.item"
)
```
It makes much more sense to treat this as a clojure source file for Quicklook purposes.

**Comment by sbarex** [comment]
Ok. I missed to handle the `com.adobe.edn` UTI. Now fixed on release 2.0.4.
